packages/material-ui/src/TablePagination/TablePaginationActions.js

```diff
@@ -47,6 +68,15 @@ const TablePaginationActions = React.forwardRef(function TablePaginationActions(
       >
         {theme.direction === 'rtl' ? <KeyboardArrowLeft /> : <KeyboardArrowRight />}
       </IconButton>
+      {showLastButton && (
+        <IconButton
+          onClick={handleLastPageButtonClick}
+          disabled={page >= Math.ceil(count / rowsPerPage) - 1}
+          aria-label="last page"
```

Comment:
We need to allow localization on the strings. See the Pagination for prior work on this problem.

Reply:
@oliviertassinari so, to allow localization, we can make ariaLabel as prop, something like
```suggestion
          aria-label={firstButtonAriaLabel}
```
which defaults to `last page`, right?

Reply:
@oliviertassinari have added support for localization